core/validation/initial_validation/schemas.py

```diff
@@ -105,3 +105,33 @@
         '"Future_High_Street_Fund".',
     ),
 ]
+
+PF_INITIAL_VAL_SCHEMA = [
+    BasicCheck(
+        sheet="Metadata",
+        column=0,
+        row=0,
+        expected_values=("1",),
+        error_message="The expected reporting period is Q3 Oct - Dec 23/24",
+    ),
+    BasicCheck(sheet="Metadata", column=0, row=1, expected_values=("4",), error_message="The expected value is V 4.0"),
+    # should this be renamed to organisation name to reflect the spreadsheet?
+    AuthorisationCheck(
+        sheet="Admin",
+        column=11,
+        row=1,
+        expected_values=(),
+        error_message="You’re not authorised to submit for {wrong_place_or_fund_type}. You can only "
+        "submit for {allowed_places_or_fund_types}.",
+        auth_type="Place Names",
+    ),
+    AuthorisationCheck(
+        sheet="Metadata",
+        column=0,
+        row=2,
+        expected_values=(),
+        error_message="You’re not authorised to submit for {wrong_place_or_fund_type}. You can only submit for "
+        "{allowed_places_or_fund_types}.",
+        auth_type="Fund Types",
+    ),
```

Comment:
I don't think an `AuthorisationCheck` on fund type is relevant for PF because unlike the TF templates where the same template can either be FHSF or TD, PF is just PF. This means we can just use a `BasicCheck` to check this field instead as the expected value does not vary.